src/main/java/dk/cngroup/lentils/service/convertors/CypherMapper.java

```diff
@@ -0,0 +1,41 @@
+package dk.cngroup.lentils.service.convertors;
+
+import dk.cngroup.lentils.dto.CypherFormDTO;
+import dk.cngroup.lentils.entity.Cypher;
+import dk.cngroup.lentils.entity.User;
+import dk.cngroup.lentils.service.UserService;
+import org.modelmapper.Converter;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Primary;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Component
+@Primary
+public class CypherMapper extends ModelMapperWrapper {
+
+    private UserService userService;
+
+    private final Converter<List<Long>, List<User>> longToOrganizerConverter =
```

Comment:
toto bych prejmenoval na: `idToOrganizerConverter` a to pod tim na `organizersToIdConverter `